src/widgets/text.rs

```diff
@@ -48,30 +50,36 @@ pub fn Text(props: TextProps) {
                         line_height.unwrap_or(size * 1.2),
                         (layout.width, layout.height),
                     );
-
                     (measurement, (layout.width, layout.height))
                 } else {
+                    should_render = false;
                     ((0.0, 0.0), (layout.width, layout.height))
                 }
             } else {
+                should_render = false;
                 ((0.0, 0.0), (0.0, 0.0))
             }
         } else {
+            should_render = false;
             ((0.0, 0.0), (0.0, 0.0))
         };
 
-    let render_command = RenderCommand::Text {
-        content: content.clone(),
-        size,
-        parent_size,
-        line_height: line_height.unwrap_or(size * 1.2),
-        font: font_name.clone().unwrap_or("Roboto".into()),
-    };
+    if should_render {
```

Comment:
This prevents rendering text until all of the conditions are met. I found this useful incases where font has loaded and we don't have the parent layout. It keeps the text from jumping around. :)